src/components/navBar/index.tsx

```diff
@@ -1,17 +1,68 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
-
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
 const RDSLogo = '/RDSLogo.png';
+const edit = '/pencil.webp';
+
+function debounce<A = unknown, R = void>(
+  fn: (args: A) => R,
+  ms: number
+): (args: A) => Promise<R> {
+  let timer: NodeJS.Timeout;
+
+  const debouncedFunc = (args: A): Promise<R> =>
+      new Promise((resolve) => {
+          if (timer) {
+              clearTimeout(timer);
+          }
+
+          timer = setTimeout(() => {
+              resolve(fn(args));
+          }, ms);
+      });
+
+  return debouncedFunc
+}
+
 
-const NavBar = () => (
+const NavBar = () => {
+  const [isEditVisible, setIsEditVisible] = useState(false);
+
+  
+
+  useEffect(() => {
+    function handleKeyDown(event:any) {
+      //alt key's keycode is 18
+      if(event.keyCode == 18){
+        setIsEditVisible(true);
+      }
+    }
+    document.addEventListener('keydown', debounce(handleKeyDown, 300));
+    
+      return function cleanup() {
+      document.removeEventListener('keydown', handleKeyDown);
+    }
```

Comment:
Should this be a custom hook?

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/278 fixed it in this PR

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/278 done it in this PR